snap/snapcraft.yaml

```
@@ -153,7 +153,8 @@ parts:
 
   freecad:
     plugin: cmake
-    source: https://github.com/FreeCAD/FreeCAD.git
+    source: https://github.com/luzpaz/FreeCAD.git
+    source-branch: PR18599
```

This doesn't look right, I can't find that branch in your fork

---

I believe this should be

```suggestion
    source: https://github.com/ein-shved/FreeCAD.git
    source-branch: about-thumbnail
```

---

:facepalm: